pipeline/velocity/main.py

```diff
@@ -9,16 +10,35 @@ def load_datasets(cfg: DictConfig) -> tuple[MyTokenizedMidiDataset, MyTokenizedM
     src_encoder = QuantizedMidiEncoder(quantization_cfg=cfg.dataset.quantization)
     tgt_encoder = VelocityEncoder()
 
-    train_translation_dataset = load_cache_dataset(
-        dataset_cfg=cfg.dataset,
-        dataset_name=cfg.dataset_name,
-        split="train",
-    )
-    val_translation_dataset = load_cache_dataset(
-        dataset_cfg=cfg.dataset,
-        dataset_name=cfg.dataset_name,
-        split="validation",
-    )
+    if "maestro" in cfg.dataset_name:
```

Comment:
We shouldn't do ifs like that - it's better to find a way to opearate on split-agnostic dataset objects.